samples/dotnet/graph-api-skills/Program.cs

```diff
@@ -179,6 +180,49 @@ public static async Task Main()
 
         await sk.RunAsync(emailMemory, outlook["SendEmailAsync"]);
 
+        Thread.Sleep(5000);
+
+        // Build email query
+        ContextVariables getEmailMessagesVariable = new();
+        getEmailMessagesVariable.Set(EmailSkill.Parameters.Select, "subject,from,receivedDateTime");
+        getEmailMessagesVariable.Set(EmailSkill.Parameters.OrderBy, "receivedDateTime DESC");
+        getEmailMessagesVariable.Set(EmailSkill.Parameters.Top, "10");
+
+        // Parse returned emails
+        SKContext emailsResult = await sk.RunAsync(getEmailMessagesVariable, outlook["GetEmailMessagesAsync"]);
+        string[] emailsSubjectOrBodyContent = emailsResult.Result.Split('\0');
+        foreach (var email in emailsSubjectOrBodyContent)
+        {
+            logger.LogInformation($"Email Subject or Body: {email}");
```

Comment:
My use case is for each email, run an additional classification skill on email attributes (e.g. email subject, email body, etc.) and then possibly either generate a auto-respond email or further actions (e.g. add reminder, etc.)

I think the consumer of this skill would want full access to a concrete Email object that encapsulates the aforementioned attributes but I understand/think the intention SKContext to to generically encapsulate the info I need in the most generic way (i.e. string) so I'm not sure what is the best way to go about my problem. Would appreciate some tips :) @craigomatic and other folks